src/ValidatorInterface.php

```diff
@@ -9,7 +9,7 @@
 /**
  * Validator validates {@link DataSetInterface} against rules set for data set attributes.
  *
- * @psalm-type RulesType = null|class-string|RuleInterface|RulesProviderInterface|object|iterable<RuleInterface|RuleInterface[]|Closure|Closure[]>
```

Comment:
Why these types were deleted?

Reply:
Is there any other way to mention them because they are more specific?

Reply:
Add to PhpDoc

Reply:
How exactly? Just as a part of the description or as a type hint?

Reply:
As you like.

Reply:
Will be fixed in #397.